view/MainMenu/MainMenu.py

```diff
@@ -0,0 +1,86 @@
+import logging
+import ipaddress
+from typing import Dict
+import pygame
+
+from view.BasicView import BasicView
+from view.MainMenu.MainMenuScreen import MainMenuScreen
+from view.MainMenu.SettingsScreen import SettingsScreen
+from view.ViewSettings import ViewSettings
+from controller.ControllerView import ControllerMainMenu
+
+
+class MainMenu(BasicView):
+
+    def __init__(self, window: pygame.display, controller: ControllerMainMenu, settings: ViewSettings) -> None:
+        super(MainMenu, self).__init__(window, controller, settings)
+
+        self.active_screens = []
+        self.mainMenuScreen = MainMenuScreen(self.window, self.controller, self, self.settings)
+        self.settingsScreen = SettingsScreen(self.window, self.controller, self, settings)
+
+        self.active_screens.append(self.mainMenuScreen)
+        logging.info("MainMenu init done")
+
+    def init(self) -> None:
+        """
+        Not yet implemented / needed
+        :return:    None
+        """
+
+    def draw(self) -> None:
+        for screen in self.active_screens:
+            screen.draw()
+
+    def receive_event(self, event: pygame.event.Event) -> None:
+        """
+        Receive Event method called by controller
+        :param event:   event
+        :return:        None
+        """
+        for screen in self.active_screens:
+            screen.receive_event(event)
+
+    def to_settings(self) -> None:
+        """
+        Interface from children view to parent
+        :return:
+        """
+        self.active_screens = [self.settingsScreen]
+
+    def to_help(self) -> None:
+        """
+        Interface from children view to parent
+        :return:
+        """
+        logging.error("Help Screen not yet implemented")
+
+    def to_main_menu(self, settings: Dict) -> None:
+        """
+        Interface from children view to parent
+        In this method the settings entered in the settingsscreen are parsed
+        :param settings:
+        :return:
+        """
+        try:
+            width, height = settings["resolution"].split("x")
+            width, height = int(width), int(height)
+            self.settings.window_height, self.settings.window_width = height, width
```

Comment:
values are set in `settings` but are not adapted by `pygame.display`

Reply:
Will not be solved in this MR.
See #51